src/engine/server/server.cpp

```diff
@@ -1463,8 +1463,8 @@ void CServer::ConStatus(IConsole::IResult *pResult, void *pUser)
 
 				CGameContext* gserver = dynamic_cast<CGameContext*>(pThis->m_pGameServer);
 
-				str_format(aBuf, sizeof(aBuf), "id=%d addr=%s name='%s' score=%d warns=%d %s", i, aAddrStr,
-					pThis->m_aClients[i].m_aName, pThis->m_aClients[i].m_Score, gserver->m_apPlayers[i]->Warnings, pAuthStr);
+				str_format(aBuf, sizeof(aBuf), "id=%d addr=%s name='%s' score=%d aimMaxDist=%f experiment=%d warns=%d %s", i, aAddrStr,
```

Comment:
Maybe better to add separate `rcd_status` command for this? Like https://github.com/ddnet/ddnet/blob/fd9178262540c96b2a99a235dce77b5be864b300/src/engine/server/server.cpp#L2282